File: common/src/main/java/com/movies/common/movie/MovieTo.java
package com.movies.common.movie;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MovieTo {
    private String id;
    private String name;
    private boolean hasAvatar;
    private LocalDateTime registered;
}

File: movies-list-service/src/main/kotlin/com/movies/list/movies/to/MovieTo.kt
package com.movies.list.movies.to

import com.movies.list.movies.MoviesGenres
import com.neovisionaries.i18n.CountryCode
import org.bson.types.Binary
import org.springframework.data.annotation.Id
import java.time.LocalDate
import java.time.LocalDateTime

data class MovieTo(
        var id: String?,

        var name: String,

        val originalName: String?,

        val hasAvatar: Boolean,

        val about: String?,

        val country: CountryCode,

        val genres: Set<MoviesGenres>,

        val premiere: LocalDate,

        val age: String,

        val time: String,

        var registered: LocalDateTime
)